DataDog/dd-trace-rb
# Backport "Reduce startup logs verbosity" to master

**What does this PR do?**

This PR backports https://github.com/DataDog/dd-trace-rb/pull/3448 "[NO-TICKET] Reduce startup logs verbosity" from the 2.0 branch to master.

**Motivation:**

This change slightly reduces the amount of logging ddtrace does (see original PR), as well as makes sure our 1.x and 2.x logging are aligned.

**Additional Notes:**

N/A

**How to test the change?**

You can use for instance

> bundle exec ruby -e "require 'ddtrace'; 2.times { Datadog.configure { } }"

to see the difference.

**For Datadog employees:**
credentials of any kind, I've requested a review from `@DataDog/security-design-and-guidance`.
